README.md

```diff
@@ -222,34 +222,19 @@ An runtime config automatically expanded by Lighter might look like
 ```
 
 ## Container Startup Sequence
-Docker images should embed the `secretary` executable. Call it at container startup to decrypt
-environment variables, before starting the actual service.
+Docker images should embed the `secretary` executable. Use it as an entrypoint
+with `secretary exec --". It will then decrypt all command line arguments and
+environment variables.
 
 *Dockerfile*
 ```
 # Install secretary
 ENV SECRETARY_VERSION x.y.z
 RUN curl -fsSLo /usr/bin/secretary "https://github.com/meltwater/secretary/releases/download/${SECRETARY_VERSION}/secretary-`uname -s`-`uname -m`" && \
     chmod +x /usr/bin/secretary
-```
-
-Container startup examples
-```
-#!/bin/sh
-set -e
-
-# Decrypt secrets
-if [ "$SERVICE_PUBLIC_KEY" != "" ]; then
-    SECRETS=$(secretary decrypt -e --service-key=/service/keys/service-private-key.pem)
```

Comment:
The exec command example doesn't support using the service key. This optional key provides more security by ensuring that the secret can only be decrypted on a specific set of machines that hold the private service key on local disk. Not sure how to solve this right off, could we have a chat about it?